helm/defectdojo/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "2.40.0-dev"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
-version: 1.6.155-dev
+version: 1.6.156-dev
```

```suggestion
version: 1.6.155-dev
```
This only gets revved on release